src/main/java/uk/gov/hmcts/reform/laubackend/cases/security/SecurityConfiguration.java

```diff
@@ -17,35 +17,19 @@
 @Order(1)
 public class SecurityConfiguration {
 
-    @Bean
-    public WebSecurityCustomizer webSecurityCustomizer() {
-        return web -> web.ignoring().requestMatchers(
-            "/swagger-ui.html",
-            "/webjars/springfox-swagger-ui/**",
-            "/swagger-resources/**",
-            "/health",
-            "/health/liveness",
-            "/health/readiness",
-            "/v2/api-docs/**",
-            "/info",
-            "/favicon.ico",
-            "/"
-        );
-    }
-
```

Comment:
Covered by SecurityFilterChain below and these were raising "deprecated" warnings.